docs/authentication/authentication.mdx

```diff
@@ -12,8 +12,8 @@ keywords:
 pagination_next: null
 pagination_prev: null
 last_update:
-  date: 08/01/2023
-  author: William May
+  date: 06/01/2023
+  author: Patricia McPhee
```

Comment:
I think this should be updated, rather than reverted

Reply:
Want to make a suggestion on this MR so we can pull in the changes? I just tapped revert on the MR so I don't have a branch locally.

Reply:
Sure